### ISSUE TITLE
Center dot for uneven pixel number screens.

### ISSUE THREAD

**Issue by Orion-Netix** [DerKuehneKollege/center-dot#6]
Hi, as title says, i have a AOC G2790VXA 27" and the 4 pixel wide dot isn't centered, a corner will take the center pixel, but it will always be offset, i maybe exagerating for some but this annoys me and i would like to know if i can make it so its a single pixel in the center of my screen. Thank you.

**Comment by DerKuehneKollege** [comment]
Good point (no pun intended), I will add this as a feature request. Idea would be to allow to set the size in a settings dialog.

**Comment by inspektorpidozra** [comment]
Yeah, that's would be nice. It's 3 px wide as I see and not complettely in the center even after adjusment. I think it will be good to have size parameter saved just in separate text file.
